app/Models/SettingsProperty.php

```diff
@@ -68,5 +68,19 @@ public function registerMediaConversions(Media $media = null): void
             ->performOnCollections('logo', 'dark_logo')
             ->height(250)
             ->keepOriginalImageFormat();
+
+        $this->addMediaConversion('portal_favicon')
+            ->format('png')
+            ->performOnCollections('portal_favicon')
+            ->height(512)
+            ->width(512)
+            ->acceptsMimeTypes([
+                'image/png',
+                'image/jpeg',
+                'image/ico',
+                'image/webp',
+                'image/jpg',
+                'image/svg',
+            ]);
```

Comment:
This is not a method for conversions and as such causes an error when you try and upload. This is for the collection.

Please define the collection in this class for `portal_favicon` and apply this and other pertinent methods such as `singleFile()` there.

https://spatie.be/docs/laravel-medialibrary/v11/working-with-media-collections/defining-media-collections

Reply:
Hi @Orrison 
Done as per requirements.
Sorry for this type of mistake. Next time i will keep this on mind.

Reply:
No problem!